src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -83,13 +83,14 @@ protected override async Task<RunResult> RunAsyncInternal(string expr, string se
                 FeatureFlags.StringInterpolation = true;
                 RecalcEngine engine;
                 RecordValue parameters;
+                var iSetup = InternalSetup.Parse(setupHandlerName);
 
-                if (setupHandlerName == "AsyncTestSetup")
+                if (string.Equals(iSetup.HandlerName, "AsyncTestSetup", StringComparison.OrdinalIgnoreCase))
                 {
                     return new RunResult(await RunVerifyAsync(expr));
                 }
 
-                if (setupHandlerName != null) 
+                if (!string.IsNullOrEmpty(iSetup.HandlerName))
```

Comment:
>IsNullOrEmpty [](http://example.com/codeflow?start=28&length=13)

nit: null check is a stronger enforcement than null or empty. So this is loosening a check. 
We should keep it strong (just null check) to simplify downstream callers,  and have test parser should fail if it's empty.

Reply:
If the handler name would be an empty, it would ignore it and execute the code normally like if there was no handler
With the null check, it will throw an exception
Either way, I'm fine but with a little preference for the first one (accepting more)